src/components/Recommend/Recommend.tsx

```diff
@@ -43,6 +43,8 @@ const Footer: React.FC<{
   shareUrl: string;
   shareQuote: string;
 }> = ({ onClickOpenModal, shareUrl, shareQuote }) => {
+  const trackLabel = 'recommendations';
```

Comment:
The category is already recommendations, consider using the name of the social network as label instead so we can have more granular information in the reports.